pom.xml

```diff
@@ -256,7 +256,7 @@
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-jdk14</artifactId>
-            <version>1.7.7</version>
+            <version>1.7.30</version>
```

Comment:
Don't do this in this PR, keep changes to what is direclty related, which this... not?

Reply:
IDK what was the reason to update it (it was not my commit :)), reverted it back